starter-code/css/main.css

```diff
@@ -6,5 +6,61 @@ light blue: #1c5380
 
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
-  margin: 0;
+  border-top: 250px solid #125688;
+  margin: 20px 20px 20px 20px;
+}
+
+#phone {
+  position: relative;
+  float: left;
+  margin-top: -170px;
+  padding: 20px;
+  padding-right: 30px;
+}
+h3 {
+  color: #1c5380;
+}
+
+#logo {
+  margin-top: -140px;
+  float: left;
+}
+
+#login {
+    position: relative;
+    top: 20px;
+    height: 35px;
+    width: 85px;
+    padding-top: 6px;
+    float: right;
+    text-align: center;
+    margin-right: 10px;
+    margin-top: -480px;
```

Comment:
-480 margin? You crazyyyy. On some mobile phones, 480 px is the entire vertical resolution, so imagine what this setting would do in that view.